napari/utils/events/_tests/test_evented_model.py

```diff
@@ -120,8 +120,8 @@ class Model(EventedModel):
 
     model = Model(values=[1, 2, 3])
 
-    # Mock events
-    model.events.values = Mock(model.events.values)
+    # MagicMock events
```

Comment:
I assume this meant `Mock` as a verb and this just got updated by a search/replace?

```suggestion
    # Mock events
```

I don't think I can stomach `MagicMock` as a verb :p

Alternatively, consider deleting the comment because it adds very little here - I already clearly see that we're mocking events (or rather the `values` emitter) from the code itself.

Reply:
lol, yup, you got me :P